.github/workflows/build_windows.yml

```diff
@@ -0,0 +1,59 @@
+name: Build (Windows)
+
+on: [push, pull_request]
+
+jobs:
+  Windows:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        configuration: ["Debug", "Release"]
+    steps:
+    - name: Define Build Options
+      shell: python
+      run: |
+        if "${{ matrix.configuration }}" == "Debug":
+          print('Configuring for Debug')
+          print('::set-env name=BUILD_PARAM::')
+          print('::set-env name=ARTIFACT_NAME::xqemu-debug')
+        else:
+          print('Configuring for Release')
+          print('::set-env name=BUILD_PARAM::--release')
+          print('::set-env name=ARTIFACT_NAME::xqemu-release')
+    - name: Clone Tree
+      uses: actions/checkout@v1
+      with:
+        fetch-depth: 1
+    - name: Install Dependencies
+      run: |
+        echo "Downloading MSYS2 environment..."
+        Invoke-WebRequest -Uri "https://github.com/mborgerson/msys2-tools/releases/latest/download/msys64.7z" -OutFile "msys64.7z"
+        echo "Extracting MSYS2 environment..."
+        7z x -y msys64.7z "-oC:\tools\"
+        echo "Updating MSYS2 environment..."
+        C:\tools\msys64\usr\bin\bash.exe -lc "pacman -Syu"
+    - name: Initialize ccache
+      id: cache
+      uses: actions/cache@v1
+      with:
+        path: C:\tools\msys64\tmp\xqemu-ccache
+        key: cache-${{ runner.os }}-${{ matrix.configuration }}-${{ github.sha }}
+        restore-keys: cache-${{ runner.os }}-${{ matrix.configuration }}-
+    - name: Compile
+      run: |
+        C:\tools\msys64\usr\bin\bash.exe -lc @"
+        set -e
+        cd `$GITHUB_WORKSPACE
+        export CCACHE_DIR=/tmp/xqemu-ccache
+        export CCACHE_MAXSIZE=250M
+        export PATH="/usr/lib/ccache/bin:/mingw64/bin:`$PATH"
+        export PATH=":`$PATH"
+        ./build.sh ${{ env.BUILD_PARAM }}
+        echo -e "\n** Compiler Cache Stats"
```

Comment:
Our other `echo` don't seem to do `\n** ` prefix. Why does this one do it?

Reply:
I like the prefix personally, but removed it for now to prevent bikeshed.

Reply:
I was just wondering why.

I'm not even sure where I could see one of the GitHub Action logs, so I have no idea how / where / why / what it puts in the output log.

Reply:
Personally I like the prefix to help differentiate our build system flow vs tool output. But this would be the first instance for now. In the future we should probably clean up the build output to make it a tad more readable, but it's fine for now.

You can see the whole workflow [here](https://github.com/mborgerson/xqemu/commit/8cdeb3de529ba2bc6fd4ccb4ba1f849a919f9480/checks?check_suite_id=325961312).

A warning: don't have Firefox open watching logs with something that spits out a lot of text to stdout. It causes my Firefox to lock up...apparently it's a known issue.

Reply:
Yes, prefix looks fine in log (and makes sense) - like I said: I was just asking as I had no idea how / where it appears in the log.
I'm fine eitherway: with or without prefix.